ui/imports/shared/views/chat/ChatTextView.qml

```diff
@@ -112,17 +112,16 @@ Item {
         }
 
         text: {
-            if(contentType === Constants.messageContentType.stickerType) return "";
+            if (contentType === Constants.messageContentType.stickerType) return "";
             let msg = Utils.linkifyAndXSS(message);
-            if(isEmoji) {
-                return StatusQUtils.Emoji.parse(msg, StatusQUtils.Emoji.size.middle);
-            } else {
-                if(isEdited){
-                    let index = msg.endsWith("code>") ? msg.length : msg.length - 4
-                    return Utils.getMessageWithStyle(StatusQUtils.Emoji.parse(msg.slice(0, index) + Constants.editLabel + msg.slice(index)), isCurrentUser, hoveredLink)
-                }
-                return Utils.getMessageWithStyle(StatusQUtils.Emoji.parse(msg), isCurrentUser, hoveredLink)
+            if (isEmoji)
+                return StatusQUtils.Emoji.parse(msg, StatusQUtils.Emoji.size.middle, StatusQUtils.Emoji.format.png);
```

Comment:
Does it really need to be png, why?

Reply:
Ok, I should read description first :facepalm:

Reply:
I've done a quick comparison:
`StatusQUtils.Emoji.parse(msg, StatusQUtils.Emoji.size.middle)` vs `StatusQUtils.Emoji.parse(msg, StatusQUtils.Emoji.size.middle, StatusQUtils.Emoji.format.png)`

![image](https://user-images.githubusercontent.com/33099791/168323394-b2465208-04b8-40a2-a6b1-a1037251ceef.png)

seems there is no difference for me :thinking:

Reply:
They are drawn big, I just modified the code so that it displays two big icons near to each other for easier comparison, that's how the original chat looks like:
![image](https://user-images.githubusercontent.com/33099791/168328495-544b5aff-e745-4770-a4f2-47a76579ed71.png)

Reply:
After a second thought on this PR, I think it would be better to not display large images as texts to avoid .svg vs .png confusion. What I propose is:
```
Loader {
   sourceComponent: isEmoji ? textEdit : svgImage
   
   Component {
       id: textEdit
       StyledTextEdit {
           ...
       }
   }
   
   Component {
       id: svgImage
       SVGImage  {
           ...
       }
   }
}
```
What do you think?